.changeset/fresh-seals-glow.md

```diff
@@ -0,0 +1,28 @@
+---
+'@prairielearn/eslint-plugin': major
+'@prairielearn/prettier-plugin-sql': major
+'@prairielearn/express-test-utils': major
+'@prairielearn/compiled-assets': major
+'@prairielearn/postgres-tools': major
+'@prairielearn/browser-utils': major
+'@prairielearn/opentelemetry': major
+'@prairielearn/signed-token': major
+'@prairielearn/named-locks': major
+'@prairielearn/migrations': major
```

Comment:
This one didn't change in this PR.

Reply:
I consolidated it from `.changeset/fair-worms-give.md`, which was deleted in this PR.